app.py

```diff
@@ -249,12 +249,18 @@ def account_post():
     """
     This function saves a user inputted game into the database
     """
-    game_name = flask.request.form.get("game_name")
+    data = flask.request.form
+    game_name = data["game_name"]
+    print(game_name)
     db_game_name = SaveGames.query.filter_by(game_name=game_name).first()
+    print(db_game_name)
     if search_game_data(game_name) == "Invalid Name":
-        flask.flash("Game does not exist or invalid name. Try again!")
-    if game_name == db_game_name:
-        flask.flash("Game already exists. Try again!")
+        flask.flash("Invalid game name. Try again!")
+        return flask.redirect(flask.url_for("account"))
+    if db_game_name:
+        flask.flash("Game already saved. Try again!")
+        return flask.redirect(flask.url_for("account"))
+
```

Comment:
"if game_name == db_game_name: " was an unnecessary line of code since all you needed to compare was the game name in the db not the game name itself.  you corrected yourself and saved yourself from writing more code 
all the routes post commit are all correct